bin/docker-helper.sh

```diff
@@ -50,7 +49,15 @@ function _fail {
 }
 
 function _get_current_version() {
-    cat ${VERSION_FILE}
+    # setuptools_scm will be installed transparently if not available, Python3 is expected to be present
+    if ! python3 -m pip -qqq list | grep -F "setuptools_scm"; then
+      python3 -m pip install -qqq setuptools_scm > /dev/null 2>&1
+    fi
+    python3 -m setuptools_scm
+}
+
+function _is_release_commit() {
+    [[ $(_get_current_version) =~ ^[0-9]+.[0-9]+.[0-9]+$ ]]
```

Comment:
```suggestion
    [[ $(_get_current_version) =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]
```
Otherwise we match `0x0x0` as well. One could argue that never happens anyway usually, but why not match it properly :)